### PR TITLE
Add EditEventCommand and EditEventCommand test classes

### PR DESCRIPTION
**Pull request by hanqinilnix**

### CODE REVIEW

**Comment by hanqinilnix** [comment]
I do not know what went wrong. I just commented the test case

File: src/test/java/seedu/address/logic/parser/schedule/EditEventCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+        // recur frequency
+        userInput = targetIndex.getOneBased() + " " + targetEventIndex.getOneBased() + RECUR_FREQUENCY_DESC_WEEKLY;
+        descriptor = new EditEventDescriptorBuilder().withRecurFrequency(VALID_RECUR_FREQUENCY).build();
+        expectedCommand = new EditEventCommand(targetIndex, targetEventIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+    }
+
+    @Test
+    public void parse_multipleRepeatedFields_acceptsLast() {
```
Comment (vishandi) [inline src/test/java/seedu/address/logic/parser/schedule/EditEventCommandParserTest.java:178]: to follow the structure in AddCommandParserTest, can consider put this inside allFieldsSpecified_success(); or use parse_multipleRepeatedFields_success() as the name.

File: src/test/java/seedu/address/logic/commands/schedule/EditEventCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        Person editedPerson = new PersonBuilder(firstPerson).withSchedule(schedule).build();
+
+        EditEventDescriptor descriptor = new EditEventDescriptorBuilder()
+                .withDate(VALID_EVENT_DATE).withRecurFrequency(VALID_RECUR_FREQUENCY).build();
+        EditEventCommand editEventCommand = new EditEventCommand(INDEX_FIRST_PERSON, lastEventIndex, descriptor);
+
+        String expectedMessage = String.format(EditEventCommand.MESSAGE_EDIT_EVENT_SUCCESS, editedEvent);
+
+        Model expectedModel = new ModelManager(new AddressBook(model.getAddressBook()), new UserPrefs());
+        expectedModel.setPerson(firstPerson, editedPerson);
```
Comment (vishandi) [inline src/test/java/seedu/address/logic/commands/schedule/EditEventCommandTest.java:83]: the only difference i noticed from this test and the previous test is in this line; `firstPerson` vs `model.getFIlteredPersonList().get(0)`

**Review by vishandi — COMMENTED** [review]
Great work!

**Pull request merged**